src/lib/api-client.js

```diff
@@ -61,6 +61,15 @@ function patch(endpoint, qs, data) {
 }
 
 //////////////////////
+export function addFault(itemAddress, faultDescription){
+
+    return post('item/fault', {
+        itemAddress,
```

Comment:
Nit: we don't really need to call it `itemAddress` since we already know it's an item